plugins/currency/zcash/zcash_flypool_hashrate_

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+
+"""
+=head1 NAME
+
+zcash_flypool_hashrate_ - Munin plugin to monitor your zcash.flypool.org hashrate (H/s)
+
+=head1 APPLICABLE SYSTEMS
+
+All systems with "python" and "munin"
+
+=head1 CONFIGURATION
+
+zcash_flypool_hashrate_<YOUR_ZCASH_ADDRESS>_<YOUR_WORKER_NAME>
+
+=head1 SYNOPSIS
+
+  ln -s /usr/share/munin/plugins/zcash_flypool_hashrate_ \
+  /etc/munin/plugins/zcash_flypool_hashrate_t1gMVWjGhdjvb71UU11JDrFmiZhgUf4x5TH_mine
+
+=head1 INTERPRETATION
+
+This plugin shows the zcash.flypool.org mining pool hashrate (H/s) of a given Zcasg address and rig name.
+Hashrate is queried via Flypool API L<https://zcash.flypool.org>.
+
+=head1 VERSION
+
+0.0.1
+
+=head1 AUTHOR
+
+L<Nils Knieling|https://github.com/Cyclenerd>
+
+=head1 LICENSE
+
+GPLv2
+
+=head1 MAGIC MARKERS
+
+ #%# family=manual
+
+=cut
+"""
+
+from __future__ import print_function
+
+import sys
+import json
+import codecs
+
+try:
+    # python3
+    from urllib.request import urlopen
+    from urllib.request import Request
+except ImportError:
+    # python2
```

Comment:
Python2 ist EOL very soon - you should not clutter the code with support for it.